server/src/relay/RelayHttpServer.go

```diff
@@ -177,8 +180,8 @@ func parseCommandLine() (relayParams librelay.RelayParams) {
 	relayHubAddress := flag.String("RelayHubAddress", "0xD216153c06E857cD7f72665E0aF1d7D82172F494", "RelayHub address")
 	defaultGasPrice := flag.Int64("DefaultGasPrice", int64(params.GWei), "Relay's default gasPrice per (non-relayed) transaction in wei")
 	gasPricePercent := flag.Int64("GasPricePercent", 10, "Relay's gas price increase as percentage from current average. GasPrice = (100+GasPricePercent)/100 * eth_gasPrice() ")
-	RegistrationBlockRate:= flag.Uint64("RegistrationBlockRate", 6000-200, "Relay registeration rate (in blocks)")
-	ethereumNodeUrl := flag.String("EthereumNodeUrl", "http://localhost:8545", "The relay's ethereum node")
+	RegistrationBlockRate := flag.Uint64("RegistrationBlockRate", 6000-200, "Relay registeration rate (in blocks)")
+	ethereumNodeURL := flag.String("ethereumNodeURL", "http://localhost:8545", "The relay's ethereum node")
```

Comment:
Note that this "refactoring" breaks the configuration API, as it changes the command-line parameter. please remove all those refactorings from the PR.

Reply:
Woah!  None of that was in this PR originally, looks like somehow my linting branch got merged in with the very, very simple fix I actually pushed in the PR.  The PR must be tracking my master branch even after the PR is submitted.  You would tend to think a PR would be tied to the HEAD as of the time of the submission.

I'll back this out and see if it tracks it again, if not I'll rebase my master branch and wait for the PR to be completed.  Sorry!